apps/nextjs-website/react-components/components/Form/Form.tsx

```diff
@@ -36,9 +51,11 @@ const Form = (props: FormProps & { onSubmit: (data: FormData) => void }) => {
     organization: '',
   });
 
+  const [selectedOption, setSelectedOption] = useState<'citizen' | 'publicEmployee' | 'techPartner' | 'developer' | 'journalist' | ''>('');
```

Comment:
I'd create a type for allowed newsletter categories instead of having to list all the options every time.

Reply:
We can append the empty string option to the above mentioned type in this specific case.